src/ui/map.js

```diff
@@ -57,7 +57,9 @@ const defaultOptions = {
 
     renderWorldCopies: true,
 
-    refreshExpiredTiles: true
+    refreshExpiredTiles: true,
+
+    redoPlacementOnPan: true
```

Comment:
I fixed this by back-porting the 300ms throttling changes from `gl-native` (this expands the scope of the change by applying the throttling to symbol re-placement in response to pitch/rotate, as well as pan).

The new code fixes a real bug in the old implementation: the re-placement timeout can't outlive the map anymore.